### PR TITLE
Blog Post: Highlights of #nostrdesign

### PR DESCRIPTION
**Pull request by dergigi**
**Final steps before merge:**

- [x] Cover image for social preview
- [x] Final review by @OpenSats/nostr-design 
- [x] Sanity check by @OpenSats/volunteers 

**Build preview:**

- ~~[/blog/nostr-design-highlights](https://os-website-git-pr-269-open-sats.vercel.app/blog/nostr-design-highlights)~~
- ~~[/blog/202405-nostr-design-highlights](https://os-website-git-pr-269-open-sats.vercel.app/blog/202405-nostr-design-highlights)~~
- [/blog/nostr-design-highlights-may](https://os-website-git-pr-269-open-sats.vercel.app/blog/nostr-design-highlights-may)

**Social preview:**

33-design-update

### CODE REVIEW

File: data/blog/nostr-design-highlights.mdx [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+authors:
+  - karnage
+  - dtonon
+  - niel
+images:
+  - /static/images/blog/33-design-update.jpg
+summary: A brief update on what the Nostr Designers have been up to.
+---
+
+The [pixel-obsessed ostriches](/nostr#nostr-design) have been busy, gently
```
Comment (karnagebitcoin) [inline data/blog/nostr-design-highlights.mdx:18]: pixel-obsessed nostriches?

**Review by karnagebitcoin — COMMENTED** [review]
Looks great! just left one comment ostriches -> nostriches

File: data/blog/nostr-design-highlights.mdx (hunk 1 above)
Comment (dtonon) [reply data/blog/nostr-design-highlights.mdx:18]: I integrate the fix in the next commit

**Review by Arvin21M — APPROVED** [review]
Nice header image (image in an image). 

Great update showing the value added to these various projects in the ecosystem by the awesome design team.

Links work and point to the right destination. 

Reads well. No showstopper changes needed.

Good to go!

🚀🚀🚀

**Pull request merged**